doc/reference/reference_sql/sql-features.rst

```diff
@@ -14,443 +13,1032 @@ else we will mark it "Fail".
+        *   -   E141-04   
+            -   Basic FOREIGN KEY constraint with the NO ACTION default
+                for both referential delete and referential update actions
+            -   ``CREATE TABLE t11 (s0 INT PRIMARY KEY, s1 INT REFERENCES t10);``
+            -   :ref:`Okay <sql_foreign_key>`.   
+        *   -   E141-06   
+            -   CHECK constraints
+            -   ``CREATE TABLE t12 (s1 INT PRIMARY KEY, s2 INT, CHECK (s1 = s2));``
+            -   Okay.  
+        *   -   E141-07   
+            -   Column defaults
+            -   ``CREATE TABLE t13 (s1 INT PRIMARY KEY, s2 INT DEFAULT -1);``   
+            -   Okay.
+        *   -   E141-08   
+            -   NOT NULL inferred on primary key
+            -   ``CREATE TABLE t14 (s1 INT PRIMARY KEY);``   
+            -   Okay. We are unable to insert NULL although we don't
+                explicitly say the column is NOT NULL.   
+        *   -   E141-10   
+            -   Names in a foreign key can be specified in any order
+            -   ``CREATE TABLE t15 (s1 INT, s2 INT, PRIMARY KEY (s1,s2));``
+                ``CREATE TABLE t16 (s1 INT PRIMARY KEY, s2 INT, FOREIGN KEY (s2,s1) REFERENCES t15 (s1,s2));``   
+            -   Okay.  
+   
+   
+E151, Transaction support   
+-------------------------
+
+..  container:: table
 
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
 
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E151-01   
+            -   COMMIT statement
+            -   ``COMMIT;``   
+            -   Fail. Tarantool supports   
+                :ref:`COMMIT <sql_commit>` but it is necessary to say 
+                :ref:`START TRANSACTION <sql_start_transaction>` first.
+        *   -   E151-02   
+            -   ROLLBACK statement
+            -   ``ROLLBACK;``
+            -   :ref:`Okay <sql_rollback>`.   
+   
+   
+E152, Basic SET TRANSACTION statement
+-------------------------------------
 
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E152-01   
+            -   SET TRANSACTION statement: ISOLATION SERIALIZABLE clause
+            -   ``SET TRANSACTION ISOLATION LEVEL SERIALIZABLE;``   
+            -   Fail. Syntax error.   
+        *   -   E152-02
+            -   SET TRANSACTION statement: READ ONLY and READ WRITE clauses
+            -   ``SET TRANSACTION READ ONLY;``   
+            -   Fail. Syntax error.   
+         
+
+E*, Other
+---------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E153   
+            -   Updatable queries with subqueries   
+            -
+            -
```

Comment:
Hello @pgulutzan,
There's no example and no test results for E153, is that correct? This table contains tests results for all other items, even cursors, though Tarantool doesn't support them and we have no examples for them.

Reply:
It is not correct, I should have given an example.
I think UPDATE "view_containing_subquery" SET column1=0; would be good enough, and the result would be Fail.
I can add it in the English manual if nobody objects.
I am changing sql-features.rst anyway because of the style guide, see
https://github.com/tarantool/doc/commit/b63189cbf4dcc955f6b0f2bd4e14c4817811c26d

Reply:
@pgulutzan this change should be performed in this PR then, as the table was refactored according to our style guides. If you can add this change to this PR, please do, if it's not convenient I can do it.
And can you please wait for us to merge this PR before finishing your work on gh-2516? This PR's almost ready, translation is completed, and I'm finishing the review

Reply:
Whoops, I see now that we've both made the same change. Okay, you have priority because I must wait for a dev person to approve  all the other changes that I made in the SQL sections. I can cancel what I did in sql-features.rst, and you can make this change in "this PR" (i.e. 2537).